tests/scripts/thread-cert/Cert_6_1_02_REEDAttach_SED.py

```diff
@@ -0,0 +1,146 @@
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[LEADER].get_state(), 'leader')
+
+        self.nodes[REED].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[REED].get_state(), 'child')
+
+        self.nodes[SED].start()
+        self.simulator.go(config.DEFAULT_CHILD_TIMEOUT + 5)
+        self.assertEqual(self.nodes[SED].get_state(), 'child')
+        self.assertEqual(self.nodes[REED].get_state(), 'router')
+
+        med_messages = self.simulator.get_messages_sent_by(SED)
+
+        # Step 2 - DUT send MLE Parent Request
+        msg = med_messages.next_mle_message(mle.CommandType.PARENT_REQUEST)
+        self.assertEqual(0x02, msg.mle.aux_sec_hdr.key_id_mode)
+        msg.assertSentWithHopLimit(255)
+        msg.assertSentToDestinationAddress("ff02::2")
+        msg.assertMleMessageContainsTlv(mle.Mode)
+        msg.assertMleMessageContainsTlv(mle.Challenge)
+        msg.assertMleMessageContainsTlv(mle.ScanMask)
+        msg.assertMleMessageContainsTlv(mle.Version)
+
+        scan_mask_tlv = msg.get_mle_message_tlv(mle.ScanMask)
+        self.assertEqual(1, scan_mask_tlv.router)
+        self.assertEqual(0, scan_mask_tlv.end_device)
+
+        # Step 4 - DUT send MLE Parent Request again
+        msg = med_messages.next_mle_message(mle.CommandType.PARENT_REQUEST)
+        self.assertEqual(0x02, msg.mle.aux_sec_hdr.key_id_mode)
+        msg.assertSentWithHopLimit(255)
+        msg.assertSentToDestinationAddress("ff02::2")
+        msg.assertMleMessageContainsTlv(mle.Mode)
+        msg.assertMleMessageContainsTlv(mle.Challenge)
+        msg.assertMleMessageContainsTlv(mle.ScanMask)
+        msg.assertMleMessageContainsTlv(mle.Version)
+
+        scan_mask_tlv = msg.get_mle_message_tlv(mle.ScanMask)
+        self.assertEqual(1, scan_mask_tlv.router)
+        self.assertEqual(1, scan_mask_tlv.end_device)
+
+        # Step 6 - DUT send Child ID Request
+        msg = med_messages.next_mle_message(mle.CommandType.CHILD_ID_REQUEST)
+        self.assertEqual(0x02, msg.mle.aux_sec_hdr.key_id_mode)
+        msg.assertSentToNode(self.nodes[REED])
+        msg.assertMleMessageContainsTlv(mle.AddressRegistration)
+        msg.assertMleMessageContainsTlv(mle.LinkLayerFrameCounter)
+        msg.assertMleMessageContainsTlv(mle.Mode)
+        msg.assertMleMessageContainsTlv(mle.Response)
+        msg.assertMleMessageContainsTlv(mle.Timeout)
+        msg.assertMleMessageContainsTlv(mle.Version)
+        msg.assertMleMessageContainsTlv(mle.TlvRequest)
+        msg.assertMleMessageContainsOptionalTlv(mle.MleFrameCounter)
+        msg.assertMleMessageContainsOptionalTlv(mle.Route64)
+
+        tlv_request_tlv = msg.get_mle_message_tlv(mle.TlvRequest)
+        self.assertEqual(mle.TlvType.ADDRESS16, tlv_request_tlv.tlvs[0])
+        self.assertEqual(mle.TlvType.NETWORK_DATA, tlv_request_tlv.tlvs[1])
+
+        # Step 10 - SED send periodic 802.15.4 Data Request messages
+        msg = med_messages.next_command_message()
```

Comment:
The next message following Child Id Request should be Data Poll for SED, otherwise something goes wrong.
Here gets next message and assert it be Data Poll?